File: engine/engine-model/src/main/scala/tools/packet/character/interaction/MiniGameClose.scala
package tools.packet.character.interaction

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MiniGameClose(private var _visitor: Boolean, private var _type: Int) extends PacketInput {
  def visitor: Boolean = _visitor

  def theType: Int = _type

  override def opcode(): SendOpcode = SendOpcode.PLAYER_INTERACTION
}

File: engine/engine-model/src/main/java/tools/packet/character/interaction/MiniGameClose.java
package tools.packet.character.interaction;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MiniGameClose(Boolean visitor, Integer theType) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PLAYER_INTERACTION;
   }
}